### PR TITLE
chore(RHINENG-6816, RHINENG-8051): PF5 migration

### PR DESCRIPTION
**Pull request by leSamo**
https://issues.redhat.com/browse/RHINENG-6816

To test:
1. make sure npm run start and npm run build scripts work
2. compare PF4 and PF5 versions behaviour and appearences

### CODE REVIEW

**Comment by bastilian** [comment]
@leSamo I tested this, and all appears to be working and looking as expected. Except for the radio-button groups, which are now vertical instead of horizontal:

Stage:
Screenshot 2024-02-07 at 11 28 59


PR:
Screenshot 2024-02-07 at 11 28 55

**Comment by leSamo** [comment]
@bastilian this is weird, when I run this PR everything is correct, did you run `rm -rf node_modules` and `npm i`? I believe this would happen if you still had PF4 packages installed.
image

**Comment by bastilian** [comment]
@leSamo Yeah, I did `rm -rf` the packages and reinstalled them. (I always do. :)):

Screenshot 2024-02-07 at 12 49 18

**Comment by leSamo** [comment]
@bastilian ah, you are using stable env, I will investigate

File: fec.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  appUrl: '/insights/registration',
+  debug: true,
+  useProxy: true,
+  proxyVerbose: true,
+  interceptChromeConfig: false,
+  plugins: [],
+  hotReload: process.env.HOT === 'true',
```
Comment (gkarat) [inline fec.config.js:8]: The same question as for malware https://github.com/RedHatInsights/malware-detection-frontend/pull/111/files#r1486147794

**Review by gkarat — APPROVED** [review]
LGTM! Thank you @leSamo

File: fec.config.js (hunk 1 above)
Comment (leSamo) [reply fec.config.js:8]: Good question, the doc you sent is out of date, `hotReload` replace `_unstableHotReload`, see:
https://github.com/RedHatInsights/frontend-components/blob/4ef5accbd645b6a416f2e2bd31096b05771056ec/packages/config/src/lib/createConfig.ts#L46

or

https://github.com/RedHatInsights/frontend-components/blob/bbef2f308320b4b879020ac43f46a23d07df0574/packages/config/src/bin/webpack.plugins.ts#L10

I'll try to create PR to fix the docs.

**Pull request merged**